apps/web/src/components/ReportModal/ReportModal.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { useTranslation } from 'next-i18next';
+
+import { Modal, Button } from 'ui';
+
+type ReportModalProps = {
```

Comment:
change to interface please, for consistency

Reply:
done